src/system/NewTabs/TabsTrigger.js

```diff
@@ -0,0 +1,50 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+const StyledTabsTrigger = props => (
+	<TabsPrimitive.TabsTrigger
+		sx={ {
+			cursor: 'pointer',
+			background: 'none',
+			mr: 3,
+			fontSize: 2,
+			px: 0,
+			pb: 3,
+			border: 'none',
+			color: 'muted',
+			transform: 'translateY(1px)',
+			'&[data-state="active"]': {
+				color: 'heading',
+				borderBottom: '1px solid',
+				borderColor: 'link',
```

Comment:
I think you can leave the focus do this job for you, you can remove the border

Reply:
I tried removing the `borderBottom` once I added the focus part but I wonder if this is the expected behavior (having the blue box)
<img width="265" alt="Screenshot 2022-11-08 at 18 44 56" src="https://user-images.githubusercontent.com/668251/200637669-fcececac-761e-4609-bd8c-7bb81db50cbf.png">

Reply:
@andrea-sdl Having the focus outline is the expected behavior, yes.

Reply:
@djalmaaraujo not removing the border since it allows to see which tabs is active when focus is not present.
I changed the code to follow the example from radix (using `box-shadow` instead of border, very nice and doesn't need `transform`)